app/lib/actions/pulp/repository/copy_units.rb

```diff
@@ -19,9 +18,9 @@ def invoke_external_task
 
           units = input[:class_name].constantize.where(:id => input[:unit_ids])
 
-          source_repo.backend_service(SmartProxy.pulp_master).copy_units(target_repo, units,
-                                                                         recursive: input[:recursive],
-                                                                         resolve_dependencies: input[:resolve_dependencies])
+          override_config = Katello::Repository.build_override_config(input)
```

Comment:
can you make this '::Katello::Repository' instead. My test raised the following error
```2019-04-03T06:04:16 [E|bac|94bdfbab] undefined method `build_override_config' for Actions::Katello::Repository:Module (NoMethodError)
```